### ISSUE TITLE
WhatsApp unexpected websocket close

### ISSUE THREAD

**Issue by KrzysztofMadejski** [42wim/matterbridge#747]
After some time following error is shown: 
```
time="2019-02-24T09:28:54-05:00" level=error msg="unexpected websocket close: websocket: close 1006 (abnormal closure): unexpected EOF" prefix=whatsapp
```

Reported by @wsdjeg on #734. As @tjird said: it is already mentioned in https://github.com/Rhymen/go-whatsapp/issues/106

**Comment by KrzysztofMadejski** [comment]
Quickfix provided by @jasontkh in https://github.com/Rhymen/go-whatsapp/issues/106#issuecomment-465915538

To make it bulletproof we should restart the connection on error.

**Comment by KrzysztofMadejski** [comment]
Blocked by https://github.com/Rhymen/go-whatsapp/issues/111 and https://github.com/Rhymen/go-whatsapp/issues/112

**Comment by KrzysztofMadejski** [comment]
- [ ] import those PRs to go-whatsapp in the matterbridge fork
- [ ] test
- [ ] release

**Comment by 42wim** [comment]
I think this can be closed in favor of #783 ?

**Comment by KrzysztofMadejski** [comment]
Agree.
